modelator/utils/apalache_jar.py

```diff
@@ -68,10 +69,25 @@ def apalache_jar_download(
     if sha256_checksum is None:
         try:
             sha256_checksum = const_values.APALACHE_SHA_CHECKSUMS[expected_version]
```

Comment:
I think we don't want to maintain `APALACHE_SHA_CHECKSUMS` if we can obtain the checksums dynamically. I would remove it and just fetch the checksum from Apalache's repo.

Reply:
Nevermind. Now I see your code below, and the fact that not all checksums are available in the repo. So let's keep `APALACHE_SHA_CHECKSUMS` as it is.